clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/deploy/description/servergroup/ServerGroupParameters.groovy

```diff
@@ -37,19 +38,31 @@ class ServerGroupParameters {
   String subnetId
   String poolId
   List<String> securityGroups
+  AutoscalingType autoscalingType
+  Scaler scaleup
+  Scaler scaledown
 
   Map<String, String> toParamsMap() {
     [
-      'flavor':instanceType,
-      'image':image,
-      'internal_port':internalPort ? internalPort.toString() : null,
-      'max_size':maxSize ? maxSize.toString() : null,
-      'min_size':minSize ? minSize.toString() : null,
-      'desired_size':desiredSize ? desiredSize.toString() : null,
-      'network_id':networkId,
-      'subnet_id':subnetId,
-      'pool_id':poolId,
-      'security_groups':securityGroups ? securityGroups.join(',') : null
+      flavor:instanceType,
```

Comment:
This seems to be the only place we don't have a space after the colon in a hashmap. Could you fix this for style?